Src/CSharpier/DocPrinter/DocPrinter.cs

```diff
@@ -187,9 +187,10 @@ private void AppendComment(LeadingComment leadingComment, Indent indent)
             //  *  keeps the * in line
             //  */
             var firstLineIndentLength =
-                firstLine!.Replace("\t", "    ").Length - firstLine.TrimStart().Length;
+                firstLine!.Replace("\t", "    ").Length
```

Comment:
This is probably worth extracting to a constant. It seems to come up a fair bit.